cis-k8s-job/values.yaml

```diff
@@ -9,4 +9,4 @@ accuknox:
   label: ""
   clusterId: ""
   tenantId: ""
-  URL: "cspm.demo.accuknox.com"
+  url: "cspm.demo.accuknox.com"
```

Comment:
@raviknox why demo env URL hardcoded in the values.yaml?

Reply:
From client's perspective, demo is what they're using.
Do you want me to refer prod env?